src/chi/components/expansion-panel/expansion-panel.scss

```diff
@@ -0,0 +1,172 @@
+@import 'mixins';
+@import 'variables';
+
+$transition-duration: 0.2s;
+
+.chi {
+
+  .m-epanel {
+
+    &__header {
+      min-height: 3.5rem;
+    }
+
+    & > .m-epanel__header {
+      display: flex;
+      margin-left: -0.25rem;
+      margin-right: -0.25rem;
+      min-height: 0rem;
+
+      & > * {
+        display: block;
+        padding-left: 0.25rem;
+        padding-right: 0.25rem;
+      }
+
+      & > .m-epanel__number {
+        width: 7.5%;
+
+        & ~ .m-epanel__title {
+          width: 27%;
+        }
+      }
+
+      & > .m-epanel__title {
+        width: 34.5%;
+      }
+
+      & > .m-epanel__content {
+        flex: 1;
+      }
+
+      & > .m-epanel__action {
+        margin-left: auto;
+      }
+    }
+
+    &__title,
+    &__number {
+      font-size: $text-larger;
+      line-height: 2rem;
+      margin: 0;
+      transition: color $transition-duration;
+
+      &.-centered {
+        text-align: center;
+      }
+    }
+
+    &.-active .m-epanel__title {
+      color: $brand-color;
+    }
+
+    &.-disabled .m-epanel__title,
+    &.-disabled .m-epanel__number {
+      color: $light-color;
+    }
+
+    &:not(.-done):not(.-done--transitioning) .-done--only,
+    &:not(.-active):not(.-active--transitioning) .-active--only {
+      display: none;
+    }
+
+    &:not(.-done) .-done--only,
+    &:not(.-active) .-active--only {
+      opacity: 0;
+    }
+
+    &__collapse {
+      position: relative;
+      transition: height $transition-duration;
+    }
+
+    &__footer {
+      align-items: center;
+      box-shadow: inset 0 1px 0 0 rgba(set-color(black), 0.12);
+      display: flex;
+      flex-direction: row;
+      min-height: 4rem;
+      padding: 1rem;
+      position: relative;
+    }
+
+    .-done--only,
+    .-active--only {
+      opacity: 1;
+      transition: opacity $transition-duration;
+    }
+
+    [data-chi-epanel-action] {
+      cursor: pointer;
+    }
+
+    &.-bordered {
+      border: .0625rem solid $border-color;
+      border-radius: $border-radius;
+      margin-bottom: 0.5rem;
+      background-color: white;
+
+      .m-epanel__header {
+        padding: 1.25rem;
+      }
+
+      .m-epanel__body {
+        border-top: .0625rem solid $border-color;
+
+        .m-epanel__content {
+          padding: 1.25rem;
+
+          & > :first-child {
+            margin-top: 0;
+          }
+
+          & > :last-child {
+            margin-bottom: 0;
+          }
+        }
+      }
+
+      & > :first-child {
+        margin-top: 0;
+      }
+
+      & > :last-child {
+        margin-bottom: 0;
+      }
+    }
+
+    &:not(.-bordered) {
+      border-bottom: .0625rem solid $border-color;
+      padding-top: 1.25rem;
+      padding-bottom: 1.25rem;
+
+      &:first-child {
+        border-top: .0625rem solid $border-color;
```

Comment:
@dani-cl-madrid let's remove the border from the first and last child to simplify this. I originally added it to the spec to help illustrate the size of each panel item but it's not necessary and we can remove.